src/main/java/com/adyen/model/testcard/TestCardRange.java

```diff
@@ -0,0 +1,142 @@
+/*
+ *                       ######
+ *                       ######
+ * ############    ####( ######  #####. ######  ############   ############
+ * #############  #####( ######  #####. ######  #############  #############
+ *        ######  #####( ######  #####. ######  #####  ######  #####  ######
+ * ###### ######  #####( ######  #####. ######  #####  #####   #####  ######
+ * ###### ######  #####( ######  #####. ######  #####          #####  ######
+ * #############  #############  #############  #############  #####  ######
+ *  ############   ############  #############   ############  #####  ######
+ *                                      ######
+ *                               #############
+ *                               ############
+ *
+ * Adyen Java API Library
+ *
+ * Copyright (c) 2020 Adyen B.V.
+ * This file is open source and available under the MIT license.
+ * See the LICENSE file for more info.
+ */
+package com.adyen.model.testcard;
+
+import com.adyen.model.ThreeDSecureData;
+import com.google.gson.annotations.SerializedName;
+
+import java.util.List;
+
+public class TestCardRange {
+    @SerializedName("address")
+    private AvsAddress address = null;
+
+    @SerializedName("cardHolderName")
+    private String cardHolderName = null;
+
+    @SerializedName("cvc")
+    private String cvc = null;
+
+    @SerializedName("expiryDates")
+    private List<ExpiryDate> expiryDates = null;
```

Comment:
the specs show an expiryMonth(enum) and expiryYear, did you generate these models from there?

Reply:
yes, `ExpiryDate` contains `expiryMonth` enum and `expiryYear`